tests/installation/welcome.pm

```diff
@@ -18,18 +18,49 @@ use testapi;
 use utils 'ensure_fullscreen';
 use version_utils qw(is_sle sle_version_at_least is_staging has_product_selection has_license_on_welcome_screen);
 
-sub run {
-    my ($self) = @_;
+sub handle_popups {
+    # See poo#19832, sometimes manage to match same tag twice and test fails due to broken sequence
+    wait_still_screen 5;
+    assert_screen(\@welcome_tags, 500);
+    # Normal exit condition
+    if (match_has_tag 'inst-betawarning' || match_has_tag 'inst-welcome' || match_has_tag 'inst-welcome-no-product-list') {
+        last;
+    }
+    if (match_has_tag 'scc-invalid-url') {
+        die 'SCC reg URL is invalid' if !get_var('SCC_URL_VALID');
+        send_key 'alt-r';    # registration URL field
+        send_key_until_needlematch 'scc-invalid-url-deleted', 'backspace';
+        type_string get_var('SCC_URL_VALID');
+        wait_still_screen 2;
+        # Press Ok to confirm scc url
+        wait_screen_change { send_key 'alt-o' };
+        next;
+    }
+    if (match_has_tag 'inst-welcome-confirm-self-update-server') {
+        wait_screen_change { send_key $cmd{ok} };
+        next;
+    }
+    if (match_has_tag 'no-product-found-on-scc') {
+        record_soft_failure 'bsc#1056413';
```

Comment:
bug has verified / fixed status so need to drop this section

Reply:
sure, but not related to this PR, could you handle this?

Reply:
sure thing if rebasing and solve conflicts is more fun for you :) 
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6145